data/language/en-GB.txt

```diff
@@ -1967,7 +1967,7 @@ STR_2763    :???
 STR_2765    :Large Tram
 STR_2766    :Win scenario
 STR_2767    :Freeze weather
-STR_2768    :Unfreeze weather
+STR_2768    :<not used anymore>
```

Comment:
We used to do this, but for the last 1-2 years we have just removed them. (Strings don't get reused anyway and leaving them out makes it easier to detect old strings in the translations.)

Reply:
Ah, I found one more such entry in the file -- that's why I went with this format. I'll remove the lot tomorrow, then.